cmd/lakefs/cmd/kv.go

```diff
@@ -0,0 +1,103 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/kv"
+	"github.com/treeverse/lakefs/pkg/logging"
+	"github.com/treeverse/lakefs/pkg/version"
+)
+
+const (
+	ScanCmdMinArgs = 1
+	ScanCmdMaxArgs = 2
+	GetCmdNumArgs  = 2
+)
+
+var kvCmd = &cobra.Command{
+	Use:    "kv",
+	Short:  "Inspect lakeFS' Key-Value Store",
+	Hidden: true,
+}
+
+var kvGetCmd = &cobra.Command{
+	Use:    "get <partition key> <key>",
+	Short:  "Return the value for the given key under the given partition",
+	Hidden: true,
+	Args:   cobra.ExactArgs(GetCmdNumArgs),
+	Run: func(cmd *cobra.Command, args []string) {
+		logger := logging.Default()
+		cfg := loadConfig()
+
+		ctx := cmd.Context()
+		logger.WithField("version", version.Version).Info("lakeFS kv")
+
+		kvParams := cfg.GetKVParams()
+		kvStore, err := kv.Open(ctx, kvParams)
+		if err != nil {
+			logger.WithError(err).Fatal("Failed to open KV store")
+		}
+		defer kvStore.Close()
+
+		val, err := kvStore.Get(ctx, []byte(args[0]), []byte(args[1]))
+		if err != nil {
+			logger.WithError(err).Fatal("Failed to get value")
+		}
+		prettyVal, err := kv.ToPrettyString(args[1], val.Value)
+		if err != nil {
+			logger.WithError(err).Fatal("Failed to build object from KV value")
+		}
+
+		fmt.Printf("%s:\n%s\n", args[1], prettyVal)
```

Comment:
Having the output in an easy to read format - or at leave option to get it in json - will enable new command to process data we fetched

Reply:
Changed to json